frontend/encore/virtual-machine.rst

```diff
@@ -92,6 +96,29 @@ connections:
     Make sure to run the development server inside your virtual machine only;
     otherwise other computers can have access to it.
 
+Docker support
+~~~~~~~~~~~~~~
+
+If you are using ``docker``, to set up a ``nodejs`` ``docker-compose`` setup
+that works with the ``localhost:8080`` domain, add this to your
+``docker-compose-yml`` file:
+
+.. code-block:: yaml
+
+    services:
+        nodejs:
+            image: node:16
+            working_dir: /var/www
+            ports:
+                - '8080:8080'
+            volumes:
+                - ./:/var/www:rw,cached
+                - ~/.symfony/certs/default.p12:/root/.symfony/certs/default.p12
+            command: bash -c "yarn && yarn run dev-server --public https://localhost:8080 --host 0.0.0.0 --https"
```

Comment:
`--https` option is deprecated.  
Maybe, it's the moment to advise using `--server-type https` instead ?

Reply:
I'll review this and make the changes! There's an issue with webpack where they have "deprecated" stuff that wasn't meant to be deprecated, so I need to double check!

Reply:
I'm confirming that your assessment here is correct!